### PR TITLE
Add convert to openvino using ovc notebook

### PR DESCRIPTION
**Pull request by as-suvorov**
CVS-117946

### CODE REVIEW

File: notebooks/121-convert-to-openvino/README.md [hunk 1]
```diff
@@ -3,26 +3,24 @@
 [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?filepath=notebooks%2F121-convert-to-openvino%2F121-convert-to-openvino.ipynb)
 [![Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/openvinotoolkit/openvino_notebooks/blob/main/notebooks/121-convert-to-openvino/121-convert-to-openvino.ipynb)
 
-This notebook shows how to convert model from original framework format to the OpenVINO IR. It describes Python conversion API and Model Optimizer command-line tool. It provides examples of Hugging Face and Pytorch models conversion to OpenVINO IR.
+This notebook shows how to convert model from original framework format to the OpenVINO IR. It describes Python conversion API and OVC command-line tool. It provides examples of Hugging Face and Pytorch models conversion to OpenVINO IR.
```
Comment (tsavina) [inline notebooks/121-convert-to-openvino/README.md:6]:
```suggestion
This notebook shows how to convert a model from its original framework format to the OpenVINO IR. It describes the Python conversion API and the OpenVINO Model Converter command-line tool. It provides examples of converting Hugging Face and PyTorch models to OpenVINO IR.
```

File: notebooks/121-convert-to-openvino/README.md [hunk 2]
```diff
@@ -3,26 +3,24 @@
...
 * Fetching example models
-* Basic conversion
-* Model conversion parameters
-  * Setting Input Shapes
-  * Cutting Off Parts of a Model
-  * Embedding Preprocessing Computation
+* Conversion
+    * Setting Input Shapes
+    * Compressing a Model to FP16
+    * Convert Models from memory
```
Comment (tsavina) [inline notebooks/121-convert-to-openvino/README.md:17]:
```suggestion
    * Converting Models from Memory
```

**Review by tsavina — APPROVED** [review]

File: notebooks/121-convert-to-openvino/121-convert-to-openvino.ipynb [hunk 3]
```diff
@@ -4,36 +4,35 @@
    "cell_type": "markdown",
```
[the remark this answers is not in the document]
Comment (as-suvorov) [reply notebooks/121-convert-to-openvino/121-convert-to-openvino.ipynb:4]: <p>Link added</p>

[the remark this answers is not in the document]
Comment (as-suvorov) [reply notebooks/121-convert-to-openvino/121-convert-to-openvino.ipynb:4]: <p>Added scale values from torchvision Imagenet preprocessing description</p>

**Review by eaidova — APPROVED** [review]

**Pull request merged**